app/frontend/src/cljs/teet/project/project_navigator_view.cljs

```diff
@@ -430,6 +430,12 @@
           :icon [icons/file-cloud-download-outlined]
           :class (<class common-styles/margin-right 1)
           :items (concat export-menu-items (default-export-menu-items project))}]
+        (when (and
+                (common-controller/feature-enabled? :vektorio)
+                (some? (:vektorio/project-id project)))
+          [common/vektorio-link {:href (common-controller/query-url :vektorio/instant-login project)
+                                 :target "_blank"}
+           (str "BIM Models" (:vektorio/project-id project))])
```

Comment:
This doesn't need a translation?

Reply:
was not mentioned in AC, I can double check